pallets/staking/src/lib.rs

```diff
@@ -1588,8 +1740,7 @@ decl_module! {
                 let extra = extra.min(max_additional);
                 ledger.total += extra;
                 ledger.active += extra;
-                let did = Context::current_identity::<T::Identity>().unwrap_or_default();
-                Self::deposit_event(RawEvent::Bonded(did, stash, extra));
```

Comment:
AFAIR this is Polymesh related change as we do want to emit and IdentityId here at the time of bonding the funds